packages/docs/.vuepress/config.js

```diff
@@ -0,0 +1,79 @@
+// .vuepress/config.js
+module.exports = {
+  theme: '@okta/vuepress-theme-nimatron',
+  dest: 'dist',
+  head: [
+    [
+      'meta', { 
+        'og:title': 'Odyssey, the Okta design system',
+        'og:type': 'article',
+        'og:url': 'https://design-docs.trexcloud.com',
+        'og:image': 'https://www.okta.com/sites/default/files/okta-logo-onblue.png',
```

Comment:
Our marketing site is known to change assets like this somewhat regularly. Is it possible to use a relative path here to avoid the possibility of a broken image link?

Reply:
fair call, fixed.